recipes/jinja_partials/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "jinja_partials" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jinja_partials-{{ version }}.tar.gz
```

Comment:
The `.tar.gz` doesn't include `requirements.txt`. 

Ideally, you'd request this be fixed upstream (probably a single-line change `MANIFEST.in`), but in the meantime:

```yaml
source:
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jinja_partials-{{ version }}.tar.gz
    sha256: ada44c441fe175d0a5d5f594b5f4c3301a1d5974d212e83031fbd8886785dd69
  - url: https://raw.githubusercontent.com/mikeckennedy/jinja_partials/v{{ version }}/requirements.txt
    sha256: 3e47955ad2dbdfe0aa8d68b57cedbd3826e3e791215ff72184aac276b55eff3b
```

Reply:
Thanks @bollwyvl! I mistakenly checked the `.tar.gz` from the Github tag and saw that `requirements.txt` is present, so I ended up changing the original source to that.